Makefile

```diff
@@ -36,11 +36,10 @@ test: venv build.stamp
 proof: venv build.stamp
 	. venv/bin/activate; mkdir -p out/ out/proof; diffenator2 proof $(shell find fonts/ttf -type f) -o out/proof
 
-images: venv build.stamp $(DRAWBOT_OUTPUT)
-	git add documentation/*.png && git commit -m "Rebuild images" documentation/*.png
+images: venv $(DRAWBOT_OUTPUT)
```

Comment:
These lines were fine, weren't they? You have to ensure the fonts are built before running the drawbot scripts, and it's good to check them in afterwards.

Reply:
There is already the `make build` command, in my experience combining these two commands causes more problems than it solves, and I almost always disable the pervious configuration. See [https://github.com/eliheuer/gf-specimen-images](https://github.com/eliheuer/gf-specimen-images) as an [example](https://github.com/eliheuer/gf-specimen-images/blob/7e861960433d09d141fcc5962e06a3553e10487f/Makefile#L39).

Anyone familiar enough with  the GFPT will know how to run `make build` before they start using `make images`. Sometime you will want to make images based on the ttfs in `fonts` directory and you might not want to rebuild because you are generating images to use in visual diffs or you want to see what the fonts look like in their current state.

If the source file is broken, the images will not build currently. 

Basically this is combining two actions that you do not always want to be combined.